tuna/miopen/utils/helper.py

```diff
@@ -39,7 +39,7 @@
 from tuna.miopen.utils.metadata import MYSQL_LOCK_WAIT_TIMEOUT
 from tuna.miopen.utils.metadata import BN_DEFAULTS
 from tuna.miopen.utils.metadata import FUSION_DEFAULTS, CONV_2D_DEFAULTS, CONV_3D_DEFAULTS
-from tuna.miopen.utils.metadata import NUM_SQL_RETRIES
+from tuna.utils.metadata import NUM_SQL_RETRIES
```

Comment:
Why we keep two metadata.py scripts? Any specific reason to create the new one?

Reply:
we have moved the common definitions to tuna.utils.metaday.py.
no two instances